docs/config-files/style-guide.md

```diff
@@ -81,15 +83,79 @@ Descriptions should be **Title Case**.
 
 Shorten labels wherever possible. E.g. `Threshold at which to send a battery report` becomes `Battery Report Threshold`.
 
+Labels should be clear and concise. They should clearly explain what the parameter does while avoiding unnnecessary technical jargon:
+
+```diff
+"paramInformation": {
+    "4": {
+-        "label": "Switch multilevel set single-activation values for pushbutton 1, Byte 1",
++        "label": "Value Sent on Pushbutton 1",
+        "valueSize": 1,
+        "minValue": 0,
+        "maxValue": 99,
+        "defaultValue": 0,
+        "unsigned": true,
+        "readOnly": false,
+        "writeOnly": false,
+        "allowManualEntry": true
+    }
+}
```

Comment:
For diffs I add 2 more spaces at the beginning of each line,
in lines that change, replace one of the spaces with +/-

Reply:
Also there's a few more tab/space weirdnesses